python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -437,6 +425,18 @@ def _update(self):
                     self.attempt_to_recover_unhealthy_nodes(now)
                 self.set_prometheus_updater_data()
 
+        # Update running nodes gauge
+        num_workers = len(self.non_terminated_nodes.worker_ids)
+        self.prom_metrics.running_workers.set(num_workers)
+
+        # Remove from LoadMetrics the ips unknown to the NodeProvider.
```

Comment:
This comment is not easy to understand. Would you mind updating the comment?

Reply:
Updated.